metis-dereference/metis-dereference-common/src/main/java/eu/europeana/metis/dereference/ProcessedEntity.java

```diff
@@ -15,7 +18,8 @@
 public class ProcessedEntity {
 
   @Id
-  private String id;
+  @JsonSerialize(using = ObjectIdSerializer.class)
```

Comment:
The input file is json? I believed it was XML? Or am I confusing with something else?

Reply:
The Vocabulary class is displayed through swagger, so for that it's definitely required.
For the ProcessedEntity we don't really display it anywhere so it could have been omitted but I added it for consistency..